src/conic_algorithm.jl

```diff
@@ -86,6 +86,7 @@ type PajaritoConicModel <: MathProgBase.AbstractConicModel
     c_new::Vector{Float64}      # Objective coefficient subvector for continuous variables
     A_new::SparseMatrixCSC{Float64,Int64} # Submatrix of A containing full rows and continuous variable columns
     keep_cols::Vector{Int}      # Indices of variables in original problem that remain after removing zeros
+    keep_var_types::Vector{Symbol} # Variable types vector on variables after transforming
```

Comment:
i use it in both mip constructor and conic constructor functions, but don't need to keep it around after that. but there are a bunch of other data variables i save but don't need after constructing (eg row_to_slckj and row_to_slckv, and all the original data) too, i just need to share them between the constructor functions